ddtrace/tracer/span.go

```diff
@@ -57,8 +57,10 @@ type span struct {
 	ParentID uint64             `msg:"parent_id"`         // identifier of the span's direct parent
 	Error    int32              `msg:"error"`             // error status of the span; 0 means no errors
 
-	finished bool         `msg:"-"` // true if the span has been submitted to a tracer.
-	context  *spanContext `msg:"-"` // span propagation context
+	finished    bool         `msg:"-"` // true if the span has been submitted to a tracer.
+	context     *spanContext `msg:"-"` // span propagation context
+	stackFrames uint         `msg:"-"` // number of stack frames
```

Comment:
I'm not sure about having these here. I'm also unsure about suggesting a good alternative. Maybe having `*config` (the tracer's config) here makes more sense.

Reply:
Please take a look and tell me if it is better now.